packages/jaeger-ui-components/src/TraceTimelineViewer/TimelineHeaderRow/TimelineCollapser.tsx

```diff
@@ -56,40 +52,31 @@ type CollapserProps = {
   onExpandAll: () => void;
 };
 
-function getTitle(value: string) {
+export default function TimelineCollapser(props: CollapserProps) {
+  const { onExpandAll, onExpandOne, onCollapseAll, onCollapseOne } = props;
   const styles = getStyles();
-  return <span className={styles.tooltipTitle}>{value}</span>;
-}
-
-export default class TimelineCollapser extends React.PureComponent<CollapserProps> {
-  containerRef: React.RefObject<HTMLDivElement>;
-
-  constructor(props: CollapserProps) {
-    super(props);
-    this.containerRef = React.createRef();
-  }
-
-  // TODO: Something less hacky than createElement to help TypeScript / AntD
-  getContainer = () => this.containerRef.current || document.createElement('div');
-
-  render() {
-    const { onExpandAll, onExpandOne, onCollapseAll, onCollapseOne } = this.props;
-    const styles = getStyles();
-    return (
-      <div className={styles.TimelineCollapser} ref={this.containerRef} data-test-id="TimelineCollapser">
-        <UITooltip title={getTitle('Expand +1')} getPopupContainer={this.getContainer}>
-          <UIIcon type="right" onClick={onExpandOne} className={cx(styles.btn, styles.btnExpanded)} />
-        </UITooltip>
-        <UITooltip title={getTitle('Collapse +1')} getPopupContainer={this.getContainer}>
-          <UIIcon type="right" onClick={onCollapseOne} className={styles.btn} />
-        </UITooltip>
-        <UITooltip title={getTitle('Expand All')} getPopupContainer={this.getContainer}>
-          <UIIcon type="double-right" onClick={onExpandAll} className={cx(styles.btn, styles.btnExpanded)} />
-        </UITooltip>
-        <UITooltip title={getTitle('Collapse All')} getPopupContainer={this.getContainer}>
-          <UIIcon type="double-right" onClick={onCollapseAll} className={styles.btn} />
-        </UITooltip>
-      </div>
-    );
-  }
+  return (
+    <div className={styles.TimelineCollapser} data-test-id="TimelineCollapser">
+      <Tooltip content="Expand +1" placement="top">
+        <span>
+          <RightIcon onClick={onExpandOne} className={cx(styles.btn, styles.btnExpanded)} />
+        </span>
+      </Tooltip>
+      <Tooltip content="Collapse +1" placement="top">
+        <span>
+          <RightIcon onClick={onCollapseOne} className={styles.btn} />
+        </span>
+      </Tooltip>
+      <Tooltip content="Expand All" placement="top">
+        <span>
+          <DoubleRightIcon onClick={onExpandAll} className={cx(styles.btn, styles.btnExpanded)} />
```

Comment:
same as before, but maybe for this icon you need to add it to `packages/grafana-ui/src/types/icon.ts`